client/src/auth/index.tsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 
 import { PageContentContainer } from "../ui/atoms/page-content";
 
+import "./index.scss";
```

Comment:
I'm sorry for being slow to review here and that's probably why my brain can't see what the change is. 

As of this change, the main CSS bundle now becomes larger. No?
Before the CSS from that `./index.scss` was only lazy-loaded as a chunk. Now, it becomes a part of the main CSS bundle which means that the (CSS) contents of that file are added to every single page even for people who don't click sign-in. 

I.e. before `.page-content-container.auth-page-container{ ...` was NOT in `client/build/static/css/main.HASH.chunk.css` but now it is.